purchase_request_department/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
This line can be removed for ascetics, not a stopper though. Also, this probably should be renamed to manifest to match current standards.